util/find-unused-errs

```diff
@@ -36,16 +36,16 @@ cd include/openssl || exit 1
 grep "$PAT" *  | grep -v ERR_FATAL_ERROR | awk '{print $3;}' | sort -u >$X1
 cd ../..
 
-for F in `cat $X1` ; do
+for F in $(cat $X1) ; do
```

Comment:
Same portability comment.  I prefer to keep the backticks.

On the other hand, a case could be made to just remove this file.

Reply:
I agree with removal of files from the repository, if they are of no use anymore.